ui/src/onboarding/containers/SigninPage.tsx

```diff
@@ -68,22 +69,28 @@ class SigninPage extends PureComponent<Props, State> {
         loading={this.state.status}
         spinnerComponent={<TechnoSpinner />}
       >
-        <FunnelPage
-          className="signin-page"
-          enableGraphic={true}
-          testID="signin-page"
-        >
-          <Panel className="signin-page--panel">
-            <Panel.Body alignItems={AlignItems.Center}>
-              <div className="signin-page--cubo" />
-              <InfluxDBCloudLogo cloud={false} className="signin-page--logo" />
-              <SigninForm />
-            </Panel.Body>
-            <Panel.Footer>
-              <VersionInfo />
-            </Panel.Footer>
-          </Panel>
-        </FunnelPage>
+        <AppWrapper>
```

Comment:
I think this needs to be `<AppWrapper type="funnel">` for layout purposes (kinda janky I know)